modules/bullet/rigid_body_bullet.cpp

```diff
@@ -888,7 +888,7 @@ void RigidBodyBullet::on_exit_area(AreaBullet *p_area) {
 	if (wasTheAreaFound) {
 		if (p_area->is_spOv_gravityPoint()) {
 			--countGravityPointSpaces;
-			assert(0 <= countGravityPointSpaces);
+			ERR_FAIL_COND(countGravityPointSpaces <= 0);
```

Comment:
This changes logic too. The asserts here and at line 869 were different, and assuming they were correct, the new `ERR_FAIL_COND` should respect that logic. `countGravityPointSapces` is initialized to 0, so I guess 0 should be a valid value.

Reply:
Oh, you are right. I missed it.

Reply:
Same as above, but this time it's the other way around.

The original condition was `countGravityPointSpaces >= 0`, so when negated, it should be `countGravityPointSpaces < 0`.

Reply:
Fixed